[2022-01-31 18:04:43,626] {taskinstance.py:903} INFO - Dependencies all met for <TaskInstance: twitter_dag.twitter_aluraonline 2022-01-29T09:00:00+00:00 [queued]>
[2022-01-31 18:04:43,637] {taskinstance.py:903} INFO - Dependencies all met for <TaskInstance: twitter_dag.twitter_aluraonline 2022-01-29T09:00:00+00:00 [queued]>
[2022-01-31 18:04:43,640] {taskinstance.py:1095} INFO - 
--------------------------------------------------------------------------------
[2022-01-31 18:04:43,640] {taskinstance.py:1096} INFO - Starting attempt 1 of 1
[2022-01-31 18:04:43,640] {taskinstance.py:1097} INFO - 
--------------------------------------------------------------------------------
[2022-01-31 18:04:43,652] {taskinstance.py:1115} INFO - Executing <Task(TwitterOperator): twitter_aluraonline> on 2022-01-29T09:00:00+00:00
[2022-01-31 18:04:43,660] {standard_task_runner.py:52} INFO - Started process 85251 to run task
[2022-01-31 18:04:43,677] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'twitter_dag', 'twitter_aluraonline', '2022-01-29T09:00:00+00:00', '--job-id', '29', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/twitter_dag.py', '--cfg-path', '/var/folders/k6/528t8tnx5llf4w8y5xx1rrm80000gn/T/tmp8174yewi', '--error-file', '/var/folders/k6/528t8tnx5llf4w8y5xx1rrm80000gn/T/tmpuyvnwa1_']
[2022-01-31 18:04:43,679] {standard_task_runner.py:77} INFO - Job 29: Subtask twitter_aluraonline
[2022-01-31 18:04:43,749] {logging_mixin.py:109} INFO - Running <TaskInstance: twitter_dag.twitter_aluraonline 2022-01-29T09:00:00+00:00 [running]> on host 1.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.ip6.arpa
[2022-01-31 18:04:43,834] {taskinstance.py:1254} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=twitter_dag
AIRFLOW_CTX_TASK_ID=twitter_aluraonline
AIRFLOW_CTX_EXECUTION_DATE=2022-01-29T09:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-01-29T09:00:00+00:00
[2022-01-31 18:04:43,849] {base.py:79} INFO - Using connection to: id: twitter_default. Host: https://api.twitter.com, Port: None, Schema: , Login: , Password: None, extra: {'Authorization': '***'}
[2022-01-31 18:04:43,852] {twitter_hook.py:36} INFO - URL: https://api.twitter.com/2/tweets/search/recent?query=AluraOnline&tweet.fields=author_id,conversation_id,created_at,id,in_reply_to_user_id,public_metrics,text&expansions=author_id&user.fields=id,name,username,created_at&start_time=2022-01-29T09:00:00.00Z&end_time=2022-01-30T09:00:00.00Z
[2022-01-31 18:04:44,399] {twitter_hook.py:36} INFO - URL: https://api.twitter.com/2/tweets/search/recent?query=AluraOnline&tweet.fields=author_id,conversation_id,created_at,id,in_reply_to_user_id,public_metrics,text&expansions=author_id&user.fields=id,name,username,created_at&start_time=2022-01-29T09:00:00.00Z&end_time=2022-01-30T09:00:00.00Z&next_token=b26v89c19zqg8o3fpe482by2mxnbz8dpqw7pxjisa7725
[2022-01-31 18:04:44,667] {twitter_hook.py:36} INFO - URL: https://api.twitter.com/2/tweets/search/recent?query=AluraOnline&tweet.fields=author_id,conversation_id,created_at,id,in_reply_to_user_id,public_metrics,text&expansions=author_id&user.fields=id,name,username,created_at&start_time=2022-01-29T09:00:00.00Z&end_time=2022-01-30T09:00:00.00Z&next_token=b26v89c19zqg8o3fpe482bx5z0ybbfbfygogyu2g66f7h
[2022-01-31 18:04:44,921] {twitter_hook.py:36} INFO - URL: https://api.twitter.com/2/tweets/search/recent?query=AluraOnline&tweet.fields=author_id,conversation_id,created_at,id,in_reply_to_user_id,public_metrics,text&expansions=author_id&user.fields=id,name,username,created_at&start_time=2022-01-29T09:00:00.00Z&end_time=2022-01-30T09:00:00.00Z&next_token=b26v89c19zqg8o3fpe4829tcg2mvif567x5j88bmezywt
[2022-01-31 18:04:45,125] {taskinstance.py:1219} INFO - Marking task as SUCCESS. dag_id=twitter_dag, task_id=twitter_aluraonline, execution_date=20220129T090000, start_date=20220131T210443, end_date=20220131T210445
[2022-01-31 18:04:45,171] {local_task_job.py:151} INFO - Task exited with return code 0
[2022-01-31 18:04:45,193] {local_task_job.py:261} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2022-02-02 10:55:59,393] {taskinstance.py:903} INFO - Dependencies all met for <TaskInstance: twitter_dag.twitter_aluraonline 2022-01-29T09:00:00+00:00 [queued]>
[2022-02-02 10:55:59,402] {taskinstance.py:903} INFO - Dependencies all met for <TaskInstance: twitter_dag.twitter_aluraonline 2022-01-29T09:00:00+00:00 [queued]>
[2022-02-02 10:55:59,402] {taskinstance.py:1095} INFO - 
--------------------------------------------------------------------------------
[2022-02-02 10:55:59,402] {taskinstance.py:1096} INFO - Starting attempt 1 of 1
[2022-02-02 10:55:59,403] {taskinstance.py:1097} INFO - 
--------------------------------------------------------------------------------
[2022-02-02 10:55:59,412] {taskinstance.py:1115} INFO - Executing <Task(TwitterOperator): twitter_aluraonline> on 2022-01-29T09:00:00+00:00
[2022-02-02 10:55:59,415] {standard_task_runner.py:52} INFO - Started process 19696 to run task
[2022-02-02 10:55:59,425] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'twitter_dag', 'twitter_aluraonline', '2022-01-29T09:00:00+00:00', '--job-id', '4', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/twitter_dag.py', '--cfg-path', '/var/folders/k6/528t8tnx5llf4w8y5xx1rrm80000gn/T/tmpnn5464h1', '--error-file', '/var/folders/k6/528t8tnx5llf4w8y5xx1rrm80000gn/T/tmpw67336yx']
[2022-02-02 10:55:59,428] {standard_task_runner.py:77} INFO - Job 4: Subtask twitter_aluraonline
[2022-02-02 10:55:59,483] {logging_mixin.py:109} INFO - Running <TaskInstance: twitter_dag.twitter_aluraonline 2022-01-29T09:00:00+00:00 [running]> on host 1.0.0.127.in-addr.arpa
[2022-02-02 10:55:59,547] {taskinstance.py:1254} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=twitter_dag
AIRFLOW_CTX_TASK_ID=twitter_aluraonline
AIRFLOW_CTX_EXECUTION_DATE=2022-01-29T09:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-01-29T09:00:00+00:00
[2022-02-02 10:55:59,568] {taskinstance.py:1463} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/Users/bezerraescossia/Github/studies/twitter-project/.venv/lib/python3.7/site-packages/airflow/models/taskinstance.py", line 1165, in _run_raw_task
    self._prepare_and_execute_task_with_callbacks(context, task)
  File "/Users/bezerraescossia/Github/studies/twitter-project/.venv/lib/python3.7/site-packages/airflow/models/taskinstance.py", line 1283, in _prepare_and_execute_task_with_callbacks
    result = self._execute_task(context, task_copy)
  File "/Users/bezerraescossia/Github/studies/twitter-project/.venv/lib/python3.7/site-packages/airflow/models/taskinstance.py", line 1313, in _execute_task
    result = task_copy.execute(context=context)
  File "/Users/bezerraescossia/Github/studies/twitter-project/airflow/plugins/operators/twitter_operator.py", line 49, in execute
    for pg in hook.run():
  File "/Users/bezerraescossia/Github/studies/twitter-project/airflow/plugins/hooks/twitter_hook.py", line 52, in run
    session = self.get_conn()
  File "/Users/bezerraescossia/Github/studies/twitter-project/.venv/lib/python3.7/site-packages/airflow/providers/http/hooks/http.py", line 72, in get_conn
    conn = self.get_connection(self.http_conn_id)
  File "/Users/bezerraescossia/Github/studies/twitter-project/.venv/lib/python3.7/site-packages/airflow/hooks/base.py", line 68, in get_connection
    conn = Connection.get_connection_from_secrets(conn_id)
  File "/Users/bezerraescossia/Github/studies/twitter-project/.venv/lib/python3.7/site-packages/airflow/models/connection.py", line 379, in get_connection_from_secrets
    raise AirflowNotFoundException(f"The conn_id `{conn_id}` isn't defined")
airflow.exceptions.AirflowNotFoundException: The conn_id `twitter_default` isn't defined
[2022-02-02 10:55:59,573] {taskinstance.py:1513} INFO - Marking task as FAILED. dag_id=twitter_dag, task_id=twitter_aluraonline, execution_date=20220129T090000, start_date=20220202T135559, end_date=20220202T135559
[2022-02-02 10:55:59,606] {local_task_job.py:151} INFO - Task exited with return code 1
[2022-02-02 10:55:59,655] {local_task_job.py:261} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-02-02 11:21:30,470] {taskinstance.py:903} INFO - Dependencies all met for <TaskInstance: twitter_dag.twitter_aluraonline 2022-01-29T09:00:00+00:00 [queued]>
[2022-02-02 11:21:30,479] {taskinstance.py:903} INFO - Dependencies all met for <TaskInstance: twitter_dag.twitter_aluraonline 2022-01-29T09:00:00+00:00 [queued]>
[2022-02-02 11:21:30,479] {taskinstance.py:1095} INFO - 
--------------------------------------------------------------------------------
[2022-02-02 11:21:30,479] {taskinstance.py:1096} INFO - Starting attempt 1 of 1
[2022-02-02 11:21:30,479] {taskinstance.py:1097} INFO - 
--------------------------------------------------------------------------------
[2022-02-02 11:21:30,489] {taskinstance.py:1115} INFO - Executing <Task(TwitterOperator): twitter_aluraonline> on 2022-01-29T09:00:00+00:00
[2022-02-02 11:21:30,491] {standard_task_runner.py:52} INFO - Started process 21942 to run task
[2022-02-02 11:21:30,499] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'twitter_dag', 'twitter_aluraonline', '2022-01-29T09:00:00+00:00', '--job-id', '6', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/twitter_dag.py', '--cfg-path', '/var/folders/k6/528t8tnx5llf4w8y5xx1rrm80000gn/T/tmpf3deo094', '--error-file', '/var/folders/k6/528t8tnx5llf4w8y5xx1rrm80000gn/T/tmp1n5zpqxx']
[2022-02-02 11:21:30,501] {standard_task_runner.py:77} INFO - Job 6: Subtask twitter_aluraonline
[2022-02-02 11:21:30,544] {logging_mixin.py:109} INFO - Running <TaskInstance: twitter_dag.twitter_aluraonline 2022-01-29T09:00:00+00:00 [running]> on host 1.0.0.127.in-addr.arpa
[2022-02-02 11:21:30,588] {taskinstance.py:1254} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=twitter_dag
AIRFLOW_CTX_TASK_ID=twitter_aluraonline
AIRFLOW_CTX_EXECUTION_DATE=2022-01-29T09:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-01-29T09:00:00+00:00
[2022-02-02 11:21:30,597] {base.py:79} INFO - Using connection to: id: twitter_default. Host: https://api.twitter.com, Port: None, Schema: , Login: , Password: None, extra: {'Authorization': '***'}
[2022-02-02 11:21:30,599] {twitter_hook.py:36} INFO - URL: https://api.twitter.com/2/tweets/search/recent?query=AluraOnline&tweet.fields=author_id,conversation_id,created_at,id,in_reply_to_user_id,public_metrics,text&expansions=author_id&user.fields=id,name,username,created_at&start_time=2022-01-29T09:00:00.00Z&end_time=2022-01-30T09:00:00.00Z
[2022-02-02 11:21:31,093] {twitter_hook.py:36} INFO - URL: https://api.twitter.com/2/tweets/search/recent?query=AluraOnline&tweet.fields=author_id,conversation_id,created_at,id,in_reply_to_user_id,public_metrics,text&expansions=author_id&user.fields=id,name,username,created_at&start_time=2022-01-29T09:00:00.00Z&end_time=2022-01-30T09:00:00.00Z&next_token=b26v89c19zqg8o3fpe482by2mxnbz8dpqw7pxjisa7725
[2022-02-02 11:21:31,368] {twitter_hook.py:36} INFO - URL: https://api.twitter.com/2/tweets/search/recent?query=AluraOnline&tweet.fields=author_id,conversation_id,created_at,id,in_reply_to_user_id,public_metrics,text&expansions=author_id&user.fields=id,name,username,created_at&start_time=2022-01-29T09:00:00.00Z&end_time=2022-01-30T09:00:00.00Z&next_token=b26v89c19zqg8o3fpe482bx5z0ybbfbfygogyu2g66f7h
[2022-02-02 11:21:31,626] {twitter_hook.py:36} INFO - URL: https://api.twitter.com/2/tweets/search/recent?query=AluraOnline&tweet.fields=author_id,conversation_id,created_at,id,in_reply_to_user_id,public_metrics,text&expansions=author_id&user.fields=id,name,username,created_at&start_time=2022-01-29T09:00:00.00Z&end_time=2022-01-30T09:00:00.00Z&next_token=b26v89c19zqg8o3fpe4829tcg2mvif567x5j88bmezywt
[2022-02-02 11:21:31,803] {taskinstance.py:1219} INFO - Marking task as SUCCESS. dag_id=twitter_dag, task_id=twitter_aluraonline, execution_date=20220129T090000, start_date=20220202T142130, end_date=20220202T142131
[2022-02-02 11:21:31,858] {local_task_job.py:151} INFO - Task exited with return code 0
[2022-02-02 11:21:31,887] {local_task_job.py:261} INFO - 1 downstream tasks scheduled from follow-on schedule check
